src/s_sound.c

```diff
@@ -940,6 +968,12 @@ void S_Init(int sfxVolume, int musicVolume)
     }
 }
 
+void S_BindSoundVariables(void)
+{
+    BIND_NUM(extra_music, EXMUS_REMIX, EXMUS_NONE, EXMUS_ROLAND_SC55,
+             "Extra soundtrack (0 = Off; 1 = Remix; 2 = Roland SC-55");
```

Comment:
Nit: Rename "Roland SC-55", `EXMUS_ROLAND_SC55`, and related vars/strings to "Original", etc.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1982/commits/e5a555ad70eaf245e3144190e061eadd49803a06